async-nats/src/jetstream/mod.rs

```diff
@@ -15,6 +15,7 @@ use crate::Client;
 
 pub mod context;
 pub mod response;
+pub mod types;
```

Comment:
Lets scope this to jetstream streams, we'll spot any duplication easily.

```suggestion
pub mod stream;
```

Reply:
I renamed the whole file. makes more sense.